_data/sidebars/home_sidebar.yml

```diff
@@ -9,10 +9,6 @@ entries:
     output: web
     folderitems:
 
-    - title: Presentation
-      url: /
-      output: web
-
     - title: Introduction
       url: /README.html
```

Comment:
Can you change this field to `url: /` as README.html would also redirect to the same page?